src/React/Redux.purs

```diff
@@ -30,25 +30,22 @@ module React.Redux
   , reducerOptic
   , spec
   , spec'
+  , composeMiddleware
   , applyMiddleware
   , fromEnhancerForeign
   ) where
 
-import Prelude (Unit, (>>=), (<<<), const, pure, id, unit)
-
+import React as React
 import Control.Monad.Eff (Eff)
 import Control.Monad.Eff.Class (class MonadEff, liftEff)
-
 import Data.Either (Either, either)
 import Data.Function.Uncurried (Fn2, Fn3, runFn2, runFn3)
 import Data.Lens (Getter', Lens', Prism', matching, set, to, view)
 import Data.Tuple (Tuple(..), fst)
-
+import Prelude (Unit, const, id, pure, unit, (<<<), (>>=))
```

Comment:
Can we put `Prelude` as the first import? I think it's pretty common amongst PureScript libraries to have this up top. We could also have it as an open import (e.g., `import Prelude`)